lib/load-json.js

```diff
@@ -66,12 +66,10 @@ const getJsonSrc = sourceObj => {
       }
     }))
   }
-  let jsonPaths
+  let jsonPaths = []
   try {
-    let testPath = fs.readdirSync(sourceObj.path)
-    if (testPath) {
-      jsonPaths = glob.sync(`${sourceObj.path}/**/*.json`)
-    }
+    fs.readdirSync(sourceObj.path)
+    jsonPaths = glob.sync(`${sourceObj.path}/**/*.json`)
```

Comment:
Is it correct that `glob.sync` is within the try block? Maybe not needed if the `readdirSync` is already being try/catched?

Reply:
`glob` doesn't throw if it doesn't find anything. But now the try/catch block has gone, this is moot